include/NetworkInterface.h

```diff
@@ -1324,7 +1322,6 @@ class NetworkInterface : public NetworkInterfaceAlertableEntity {
   };
   void luaUsedPorts(lua_State *vm) { usedPorts.lua(vm, this); };
 
-  void getVLANHostsPorts(lua_State *vm);
```

Comment:
This function was removed but it's still used in pro/scripts/lua/rest/v2/get/host/hosts_details_by_port_filters.lua

Reply:
In the PRO repository, there's another PR where I also modified the hosts_details_by_port_filters.lua REST removing the usage of that function.